src/cbc_sdk/enterprise_edr/threat_intelligence.py

```diff
@@ -254,17 +421,66 @@ def reports(self):
 
         return reports_
 
+    def add_report_ids(self, report_ids):
+        """
+        Adds new report IDs to the watchlist.
+
+        Args:
+            report_ids (list[str]): List of report IDs to be added to the watchlist.
+        """
+        old_report_ids = self.report_ids if self._info.get('report_ids') else []
+        self.update(report_ids=(old_report_ids + report_ids))
+
+    def add_reports(self, reports):
+        """
+        Adds new reports to the watchlist.
+
+        Args:
+            reports (list[Report]): List of reports to be added to the watchlist.
+        """
+        report_ids = []
+        for report in reports:
+            report.validate()
+            if report._from_watchlist:
+                report_ids.append(report._info['id'])
+        self.add_report_ids(report_ids)
+
 
 class Feed(FeedModel):
     """Represents an Enterprise EDR feed's metadata."""
     urlobject = "/threathunter/feedmgr/v2/orgs/{}/feeds"
     urlobject_single = "/threathunter/feedmgr/v2/orgs/{}/feeds/{}"
     primary_key = "id"
     swagger_meta_file = "enterprise_edr/models/feed.yaml"
-
-    @classmethod
-    def _query_implementation(self, cb, **kwargs):
-        return FeedQuery(self, cb)
+    SCHEMA_IOCV2 = Schema(
```

Comment:
Can we move these from `class Feed` to `class FeedModel`? The rest of the validation code can stay as is just the schemas in the base class makes more sense to me

Reply:
Personally, I think it makes _less_ sense, but, if you insist, it shall be done.

Reply:
Thanks! The reason is the Watchlist class inherits FeedModel not Feed so this is a future looking request in case we want to consume them there too